WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/Create Shipping Label Form/Shipping Address Validation/ShippingLabelAddressFormViewController.swift

```diff
@@ -52,8 +52,14 @@ final class ShippingLabelAddressFormViewController: UIViewController {
 
     /// Init
     ///
-    init(siteID: Int64, type: ShipType, address: ShippingLabelAddress?, completion: @escaping Completion) {
-        viewModel = ShippingLabelAddressFormViewModel(siteID: siteID, type: type, address: address)
+    init(
+        siteID: Int64,
+        type: ShipType,
+        address: ShippingLabelAddress?,
+        validationError: ShippingLabelAddressValidationError?,
```

Comment:
Just out of curiosity, is there a reason why we wouldn't give this parameter a default `nil` value? I was thinking that would simplify things so we wouldn't have to explicitly set it to `nil` in the various places where the address is successfully validated.

Reply:
Not really. I think I initially did that, but then it felt a bit better to do it explicitly. In the end I didn't have a strong preference towards either option in this case.